CoreFoundation/Locale.subproj/CFCalendar.c

```diff
@@ -422,6 +422,7 @@ static UCalendarDateFields __CFCalendarGetICUFieldCodeFromChar(char ch) {
     case 'G': return UCAL_ERA;
     case 'y': return UCAL_YEAR;
     case 'M': return UCAL_MONTH;
+    case 'l': return UCAL_IS_LEAP_MONTH;
```

Comment:
This is something we are going to have to verify since it will upstream into CoreFoundation on darwin.